tables/Supplemental-table-1-modality-overview.tsv

```diff
@@ -0,0 +1,21 @@
+scpca_project_id	Diagnosis group	Total number of libraries	Single-cell	Single-nucleus	Bulk RNA	With CITE-seq	Spatial transcriptomics	With cell hashing	Diagnoses
```

Comment:
My only comment is that I would move `Diagnoses` immediately after `Diagnosis group`